packages/lodestar/src/tasks/index.ts

```diff
@@ -69,8 +69,8 @@ export class TasksService implements IService {
   };
 
   private handleFinalizedCheckpointChores = async (finalized: BlockSummary, pruned: BlockSummary[]): Promise<void> => {
-    new ArchiveBlocksTask(this.config, {db: this.db, logger: this.logger}, finalized, pruned).run();
-    new ArchiveStatesTask(this.config, {db: this.db, logger: this.logger}, finalized, pruned).run();
+    await new ArchiveBlocksTask(this.config, {db: this.db, logger: this.logger}, finalized, pruned).run();
+    await new ArchiveStatesTask(this.config, {db: this.db, logger: this.logger}, finalized, pruned).run();
```

Comment:
can we do Promise.all here?

Reply:
I want to order it so that when we restart from last known state, we're sure finalized blocks stay ArchiveBlock db already

Reply:
If the node exits cleanly, I don't think it shouldn't matter. I think the node.stop would await any open promises.